UsabillaBindings/Xamarin.Usabilla.iOS/UsabillaXamarin.cs

```
@@ -1,18 +1,174 @@
+        public void ShowFeedbackFormWithScreenshoot(string formId, Action<XUFormLoadResult> result)
+        {
+            aDelegate.Result = result;
+            var window = UIApplication.SharedApplication.KeyWindow;
+            var vc = window.RootViewController.View;
+            UIImage image = UsabillaIos.Usabilla.TakeScreenshot(vc);
+            UsabillaIos.Usabilla.LoadFeedbackForm(formId, image);
+
+        }
+        public IDictionary<string, string> CustomVariables
+        {
+            get
+            {
+                Dictionary<string, string> dictionary = new Dictionary<string, string>();
+                foreach (KeyValuePair<NSObject, NSObject> entry in UsabillaIos.Usabilla.CustomVariables)
+                {
+                    dictionary.Add(entry.Key.ToString(), entry.Value.ToString());
+                }
+                return dictionary;
+            }
+            set
+            {
+                NSMutableDictionary<NSString, NSObject> dictionary = new NSMutableDictionary<NSString, NSObject>();
+                foreach (KeyValuePair<string, string> entry in value)
+                {
+                    NSString val = new NSString(entry.Value);
+                    NSObject obj = NSObject.FromObject(val);
+                   dictionary.Add(new NSString(entry.Key), obj);
+                }
+
+                NSDictionary<NSString, NSObject> aDictionary = new NSDictionary<NSString, NSObject>(dictionary.Keys, dictionary.Values);
+                if (aDictionary != null)
+                {
+                    UsabillaIos.Usabilla.CustomVariables = aDictionary;
+                }
+
+            }
+        }
+
+        public bool Dismiss()
         {
-            Console.WriteLine("Initializing SDK for iOS");
+            return UsabillaIos.Usabilla.Dismiss();
         }
+
+        public void SetDataMasking(IList<string> masks, char maskCharacter)
+        {
+            string[] stringArray = null;
+            if (masks != null)
```

In Android when the masks is null, it uses the default masks. We should align on the behavior.

---

Adde default parameters, as discussed to have same Xamarin implementation, but it is handle by the native iOS in the same way